Description: Tune build-in features at uwsgiconfig.py
 By default, some features of uWSGI are disabled in uwsgiconfig.py. This patch
 tunes uwsgiconfig.py for building-in all possible features of uWSGI.
 .
 Enable following features:
   * SCTP support
   * Erlang integration
   * Profiler
   * support of configuration via LDAP
   * WSGI2 support
   * Routing system
 .
 Embed following plugins:
   * Lua plugin
   * PSGI plugin
   * Rack plugin
 .
 Also, define directory where packaged plugins are installed.
Author: Leonid Borisenko <leo.borisenko@gmail.com>
Forwarded: not-needed
Last-Update: 2011-01-18

Index: uwsgi/uwsgiconfig.py
===================================================================
--- uwsgi.orig/uwsgiconfig.py	2011-01-31 21:15:14.000000000 +0200
+++ uwsgi/uwsgiconfig.py	2011-01-31 21:35:15.000000000 +0200
@@ -4,15 +4,15 @@
 INI=True
 YAML=True
 SNMP=True
-SCTP=False
-ERLANG=False
+SCTP=True
+ERLANG=True
 SPOOLER=True
 EMBEDDED=True
 UDP=True
 MULTICAST=True
 THREADING=True
 SENDFILE=True
-PROFILER=False
+PROFILER=True
 NAGIOS=True
 PROXY=True
 PASTE=True
@@ -21,18 +21,18 @@
 UGREEN=True
 HTTP=True
 EVDIS=False
-LDAP=False
-WSGI2=False
-ROUTING=False
+LDAP=True
+WSGI2=True
+ROUTING=True
 STACKLESS=False
 #PLUGINS = ['psgi']
-PLUGINS = []
+PLUGINS = ['lua', 'psgi', 'rack']
 USWALLOW=False
 UNBIT=False
 DEBUG=False
 EMBED_PLUGINS=True
 UWSGI_BIN_NAME = 'uwsgi'
-UWSGI_PLUGIN_DIR = '.'
+UWSGI_PLUGIN_DIR = '/usr/lib/uwsgi/plugins'
 
 
 # specific compilation flags
